src/components/asset-list/RecyclerAssetList2/FastComponents/FastCurrencySelectionRow.tsx

```diff
@@ -0,0 +1,334 @@
+            {showBalance && (
+              <View style={[cx.column, { height: 34 }]}>
+                <Text align="right" size="16px">
+                  {item?.native?.balance?.display ??
+                    `${nativeCurrencySymbol}0.00`}
+                </Text>
+                <Text
+                  align="right"
+                  color={{ custom: theme.colors.blueGreyDark50 }}
+                  size="14px"
+                >
+                  {item?.balance?.display ?? ''}
+                </Text>
+              </View>
+            )}
+          </View>
+        </View>
+      </ButtonPressAnimation>
+      {!showBalance && (
+        <View style={[cx.fav]}>
+          {(!item?.isNativeAsset || isETH(item?.address)) && !showBalance && (
+            <ContextMenuButton
+              activeOpacity={0}
+              isMenuPrimaryAction
+              useActionSheetFallback={false}
+              wrapNativeComponent={false}
+              {...contextMenuProps}
+            >
+              <ButtonPressAnimation>
+                <SafeRadialGradient
+                  center={[0, 15]}
+                  colors={colors.gradients.lightestGrey}
+                  style={[cx.gradient, cx.igradient]}
+                >
+                  <Text
+                    color={{ custom: colors.alpha(colors.blueGreyDark, 0.3) }}
+                    weight="bold"
+                  >
+                    􀅳
+                  </Text>
+                </SafeRadialGradient>
+              </ButtonPressAnimation>
+            </ContextMenuButton>
+          )}
+          {showFavoriteButton && (
+            <ButtonPressAnimation onPress={toggleFavorite}>
+              <SafeRadialGradient
+                center={[0, 15]}
+                colors={
+                  favorite
+                    ? [
+                        colors.alpha('#FFB200', isDarkMode ? 0.15 : 0),
+                        colors.alpha('#FFB200', isDarkMode ? 0.05 : 0.2),
+                      ]
+                    : colors.gradients.lightestGrey
+                }
+                style={[cx.gradient, cx.starGradient]}
+              >
+                <RNText
+                  ellipsizeMode="tail"
+                  numberOfLines={1}
+                  style={[cx.star, favorite && cx.starFavorite]}
+                >
+                  􀋃
+                </RNText>
+              </SafeRadialGradient>
+            </ButtonPressAnimation>
+          )}
+          {showAddButton && (
+            <ButtonPressAnimation onPress={toggleFavorite}>
+              <SafeRadialGradient
+                center={[0, 15]}
+                colors={colors.gradients.lightestGrey}
+                style={[cx.gradient, cx.addGradient]}
+              >
+                <RNText style={cx.addText}>+</RNText>
+              </SafeRadialGradient>
+            </ButtonPressAnimation>
+          )}
+        </View>
+      )}
+    </View>
+  );
+});
+
+const cx = StyleSheet.create({
```

Comment:
Since we already have `sx` variable in the codebase we decided to use `sx` everywhere - all the other PRs did move to `sx` too.